README.md

```diff
@@ -82,6 +82,11 @@ Inputs (all of them required):
 - `repository-list` 
 - `tool-list`
 
+Optional inputs: 
+
+- `report_level`: all|warn|error (default: all)
+- `fail_level`: warn|error (default: warn)
```

Comment:
I think the default should be error. If there are instances that are clearly bugs we should elevate them in the linter to error.

Reply:
My rationale was that warnings correspond to violations of best practices which we usually do not want. But we can also do this per repo (I would definitely suggest this for the IUC).